### ISSUE TITLE
Feature request: Show code coverage as a percentage

### ISSUE THREAD

**Issue by rappie** [crytic/echidna#787]
_Must have_
Being able to know echidna's code coverage as a percentage

_Should have_
Write the coverage percentage to a file in the corpus directory

_Would be nice_
Show a realtime percentage in the echidna UI

**Comment by 0xPhaze** [comment]
Agree, this information would be great to have and could be used to perform additional analysis. On top of that I would say that having the absolute number of lines / opcodes that were covered is also helpful.

**Comment by gustavo-grieco** [comment]
@elopez I think the new reports are already doing this, right?

**Comment by gustavo-grieco** [comment]
Indeed, we have percentages there. Closing this one.
